src/gflownet/tasks/seh_frag.py

```diff
@@ -60,10 +60,10 @@ def __init__(self, dataset: Dataset, temperature_distribution: str, temperature_
         self.temperature_sample_dist = temperature_distribution
         self.temperature_dist_params = temperature_parameters
 
-    def _flat_reward_transform(self, y: Union[float, Tensor]) -> FlatRewards:
+    def flat_reward_transform(self, y: Union[float, Tensor]) -> FlatRewards:
```

Comment:
Are these `flat_reward_transform` changes needed for sqlite logging? If not, it is poor practice to slip them into a different PR. Better to separate into two PRs.

Reply:
True, I'll be more careful next time!